src/main.tsx

```diff
@@ -1,7 +1,7 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
 import './index.css'
-import App from './App.jsx'
+import App from './App.tsx'
```

Comment:
TS5097: An import path can only end with a .tsx extension when allowImportingTsExtensions is enabled.